implementation/sequicity/tsd_net.py

```diff
@@ -234,7 +319,12 @@ def forward(self, u_enc_out, z_tm1, last_hidden, u_input_np, pv_z_enc_out, prev_
             proba = torch.cat([proba, u_copy_score[:, cfg.vocab_size:]], 1)
         else:
             # IHMO also compute copy probability from the encoder output from the previous time step
-            # and concat it 
+            # and concat it.
+            #
+            # This sounds interensting. Seems like an additional trick. Is this
```

Comment:
I don't think it is mentioned in the paper, however I am also not sure if I understood it fully in the implementation.

Reply:
Let's skip it for now and not think about it. I think it's of low priority. Maybe "nice to do" after everything else works.